android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -327,6 +328,36 @@ private void setNavHeader(Event event) {
             OpenEventApp.picassoWithCache.load(R.mipmap.ic_launcher).into(headerView);
         }
     }
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu_extras, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        displayCopyrightInformation();
```

Comment:
Please add a switch here with selecting id of copyright only

Reply:
@iamareebjamal done sir please have a look